tests/test_shipment.py

```diff
@@ -175,3 +175,44 @@ def test_rerate(vcr):
     new_rate_id = shipment.rates[0].id
     assert new_rate_id is not None
     assert new_rate_id != rate_id
+
+
+@pytest.mark.vcr()
+def test_smartrate(vcr):
+    to_address = {
+        'name': 'Dr. Steve Brule',
+        'street1': '179 N Harbor Dr',
+        'city': 'Redondo Beach',
+        'state': 'CA',
+        'zip': '90277',
+        'country': 'US',
+        'phone': '4153334444',
+        'email': 'dr_steve_brule@gmail.com'
+    }
+    from_address = {
+        'name': 'EasyPost',
+        'street1': '417 Montgomery Street',
+        'street2': '5th Floor',
+        'city': 'San Francisco',
+        'state': 'CA',
+        'zip': '94104',
+        'country': 'US',
+        'phone': '4153334444',
+        'email': 'support@easypost.com'
+    }
+    parcel = {
+        'length': 20.2,
+        'width': 10.9,
+        'height': 5,
+        'weight': 65.9
+    }
+    shipment = easypost.Shipment.create(
+        to_address=to_address,
+        from_address=from_address,
+        parcel=parcel,
+    )
+
+    assert shipment.rates
+
+    smartrates = shipment.get_smartrates()
+    assert smartrates['result'][0]['time_in_transit']
```

Comment:
what about maybe having this test assert more than just the non-nullity of this field?

Reply:
Fair, I've beefed up the assertions of the test.